docs/application/web/api/3.0/device_api/mobile/tizen/cordova/media.html

```diff
@@ -876,7 +876,7 @@ <h4>Methods</h4>
 </div>
 <h2 id="api-features">2. Related Feature</h2>
 <div id="def-api-features" class="def-api-features">
-        Method <a href="systeminfo.html#SystemInfo::getCapability">tizen.systeminfo.getCapability()</a> can be used in application runtime to check whether this API is supported.
+        Method <a href="../systeminfo.html#SystemInfo::getCapability">tizen.systeminfo.getCapability()</a> can be used in application runtime to check whether this API is supported.
```

Comment:
Question.

Indeed, we don't need to apply same fix to both master and tizen_5.5_prepare branches,
as in the future the fixes in tizen_5.5_prepare will all be merged into master.

As these fixes are already applied to master, you don't need to do the same.

The problem is this : 
&nbsp;&nbsp;&nbsp;Aren't these broken links in Tizen API are same in Tizen **5.5** API files?

For example we have same sentence with this in https://docs.stg.tizen.org/staging/885/application/web/api/5.5/device_api/mobile/tizen/cordova/media.html#api-features and it causes broken link error.

I think the files which need to be fixed are only Tizen 5.5 Web API files, as others are already fixed on master.

Would you please check?

Reply:
Yes, we understand that. We are working on finding dead URLs, however, applying this change now rather than later makes it easier.

Reply:
I see. So, does merging this PR helps you to your work? Or are you going to update this PR for actual fix??

Reply:
I think it shouldn't do any harm to merge it. I am working on a semi-automated script for finding dead links. This process is a time-consuming task, so having fewer dead links will indeed make it easier.